APMrover2/defines.h

```diff
@@ -59,6 +59,7 @@ enum GuidedMode {
 #define LOG_SONAR_MSG 		    0x07
 #define LOG_ARM_DISARM_MSG      0x08
 #define LOG_STEERING_MSG        0x0D
+#define LOG_GUIDEDTARGET_MSG    0x0E
```

Comment:
We would need to do that for all vehicle types as I'd like to keep the code as consistent across all vehicles as possible.  As I'm trying to get Rover released I'll skip this for the moment but if you wanted to submit a PR with that change I think that would be fine.